neptune/alpha/internal/init_impl.py

```diff
@@ -66,7 +67,11 @@ def init(
         raise ValueError('connection_mode should be one of ["async", "sync", "offline"]')
 
     project_obj = backend.get_project(project)
-    exp = backend.create_experiment(project_obj.uuid)
+    if experiment:
+        exp, initial_attributes = backend.get_experiment_with_attributes(project, experiment)
```

Comment:
Could we just call sync on experiment after fetching it from server?
I think it's necessary to have very lightweight `getExperiment` (without attributes) endpoint on server side anyway, so it won't be so bad to have two calls instead of one.
Implement it if you agree.